src/modules/commandrunner/src/lib/CommandRunner.cpp

```diff
@@ -481,7 +516,7 @@ int CommandRunner::LoadPersistedCommandStatus(const std::string& clientName)
     int status = 0;
 
     std::lock_guard<std::mutex> lock(m_diskCacheMutex);
-    std::ifstream file(CommandRunner::PERSISTED_COMMANDSTATUS_FILE);
+    std::ifstream file(PERSISTED_COMMANDSTATUS_FILE);

```

Comment:
>PERSISTED_COMMANDSTATUS_FILE [](http://example.com/codeflow?start=23&length=28)

Change to const char* and members of CommandRunner class. For example:

CommandRunner::persistedCommandStatusFile